### PR TITLE
User profile Changes for v1.0

### PR DESCRIPTION
**Pull request by tshradheya**
Please suggest any more changes we can make

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
 See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-by-step solution.
 
 [appendix]
+== Target Users
+
+This application is targeted towards University students, aiming to meet their needs throughout their University life, and hopefully beyond. University students meet a lot of people in the university, such as project group members, tutors, lecturers, friends and other connections. Nowadays it is important for students to build connections with other people, as connections may also help students for their future career after graduation. Since the students meet a lot of people, they may not be able to remember all the information about the people they met. Therefore, iContacts aims to help students to store and manage their contacts easily.
```
Comment (chuaweiwen) [inline docs/DeveloperGuide.adoc]: "...aiming to meet their needs throughout their university life..." (change 'University' to lower case 'university')

"University students meet a lot of people in the university, such as project group members, tutors, lecturers, friends and other connections."
Students meet **many** people in the university, such as tutors, lecturers, friends, peers, project group members and other connections.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -761,41 +775,49 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* *` |user |find my link:#frequently-used-contacts[frequently used contacts] at the top of the displayed list when I do a search |quickly find the contact information I need
 
-|`* *` |user |see frequently used contacts instantly |locate them more easily
+|`* *` |regular user|see frequently used contacts instantly |locate them more easily
```
Comment (chuaweiwen) [inline docs/DeveloperGuide.adoc]: For consistency purposes, put another space before the "|"
Like "|regular user|" becomes "|regular user |"

Same applies for line 784.

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
 See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-by-step solution.
 
 [appendix]
+== Target Users
+
+This application is targeted towards University students, aiming to meet their needs throughout their university life, and hopefully beyond. University students meet many people in the university, such as project group members, tutors, lecturers, friends and other connections. Nowadays it is important for students to build connections with other people, as connections may also help students for their future career after graduation. Since the students meet a lot of people, they may not be able to remember all the information about the people they met. Therefore, iContacts aims to help students to store and manage their contacts easily.
+
+*  User Profiles
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc:721]: User Profile
I don't think there is a 's' at the back

File: docs/DeveloperGuide.adoc [hunk 4]
```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
...
+
+**  University Student
+**  Student part of many CCA clubs
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc:724]: ...and committees

File: docs/DeveloperGuide.adoc [hunk 5]
```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
...
+**  Student doing the role of a Tutor
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc:725]: Student taking up the role of Teaching Assistant/Tutor

File: docs/DeveloperGuide.adoc [hunk 6]
```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
...
+**  Student who is interning and has colleagues
+**  Student having close friends and family
+
```
Comment (justinpoh) [inline docs/DeveloperGuide.adoc:728]: Business students that need to frequently share contacts?
lol I am not so sure about this...just a random example

**Pull request merged**